ethminer/MinerAux.h

```diff
@@ -1077,14 +1091,12 @@ class MinerCLI
 	unsigned m_openclDeviceCount = 0;
 	unsigned m_openclDevices[16];
 	unsigned m_openclThreadsPerHash = 8;
-#if !ETH_ETHASHCUDA
-	unsigned m_globalWorkSizeMultiplier = CLMiner::c_defaultGlobalWorkSizeMultiplier;
-	unsigned m_localWorkSize = CLMiner::c_defaultLocalWorkSize;
-#endif
 #endif
+unsigned m_globalWorkSizeMultiplier_CL = CLMiner::c_defaultGlobalWorkSizeMultiplier;
```

Comment:
Again, these would make more sense if the names lined up with the constant names.
Something like:

unsigned m_globalGridSize = CUDAMiner::c_defaultGridSize;